### PR TITLE
Milestone 4

### PR DESCRIPTION
**Pull request by ifaith1**

### CODE REVIEW

File: Assets/Scripts/Door.cs [hunk 1]
```diff
@@ -5,9 +5,49 @@
...
+
+    [Tooltip("Play this audio clip when the player interacts with a locked door without owning the key")]  
+    [SerializeField]
+    private AudioClip lockedAudioClip;
+
+    [Tooltip("Play this audio clip when the player opens the door")]
+    [SerializeField]
+    private AudioClip openAudioClip;
+
+    //public override string DisplayText => isLocked ? lockedDisplayText : base.DisplayText;
```
Comment (coffeeanddraw) [inline Assets/Scripts/Door.cs:28]: Delete unused code

File: Assets/Scripts/InventoryMenuItemToggle.cs [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+    /// <summary>
+    /// This will be plugged into the toggle's "OnValueChanged" property in the editor
+    /// and called whenever the toggle is clicked
+    /// </summary>
+
+    public void InventoryMenuItemWasToggled(bool isOn)
+    {
+        // We only want to do the stuff when the toggle has been selected. Not when it has been deselected.
+        //if (isOn)
```
Comment (coffeeanddraw) [inline Assets/Scripts/InventoryMenuItemToggle.cs:34]: delete commented duplicate code

**Review by coffeeanddraw — APPROVED** [review]
Codes look good. Delete a couple commented out lines of code.

File: Assets/Scripts/Door.cs (hunk 1 above)
Comment (ifaith1) [reply Assets/Scripts/Door.cs:28]: Unused code has been deleted!

File: Assets/Scripts/InventoryMenuItemToggle.cs (hunk 2 above)
Comment (ifaith1) [reply Assets/Scripts/InventoryMenuItemToggle.cs:34]: Duplicated code has been deleted

**Pull request merged**